File: Dev/PythonServer/structure/cab/cabrequest.py
from locvertex import LocVertex
class CabRequest:
    def __init__(self):
        self.area = ''
        self.location = LocVertex()
    def parsing(self,value):
#        print 'Request parsing'
        if 'area' in value:
            self.area = value['area']
        if 'location' in value:
            self.location.parsing(value['location'])

File: Dev/Android/Codes/Prod/MyApplication/app/src/main/java/com/example/quentin/myapplication/structures/taxi/CabRequest.java
package com.example.quentin.myapplication.structures.taxi;

public class CabRequest {

    private String area;
    private LocVertex location;

    // Default constructor
    public CabRequest() {
        area="";
        location=new LocVertex();
    }

    @Override
    public String toString() {
        return "CabRequest [area=" + area + ", location=" + location + "]";
    }

    //************ GETTERS *************//

    public String getArea() {
        return area;
    }

    public LocVertex getLocation() {
        return location;
    }

    //***************** SETTERS ***************//

    public void setArea(String area) {
        this.area = area;
    }

    public void setLocation(LocVertex location) {
        this.location = location;
    }
}
